File: Exemplos/App_eXperience_XamarinAndroid/Elgin_AppExperience_XamarinAndroid-font/NFCE/It4rAbstract.cs
using BR.Com.Daruma.Framework.Mobile;

namespace M8
{
    public abstract class It4rAbstract
    {
        protected DarumaMobile dmf;

        protected It4rAbstract(DarumaMobile dmfObject)
        {
            dmf = dmfObject;
        }

        public abstract void VenderItem(string descricao, string valor, string Codigo);

        public abstract void EncerrarVenda(string valorTotal, string formaPagamento);

        public abstract void ConfigurarXmlNfce();
    }
}

File: Exemplos/App_eXperience_ReactNative/Elgin_AppExperience_ReactNative-font/android/app/src/main/java/com/elginm8/NFCE/It4rAbstract.java
package com.elginm8.NFCE;

import androidx.annotation.NonNull;

import br.com.daruma.framework.mobile.DarumaMobile;

public abstract class It4rAbstract {
    protected DarumaMobile dmf;

    protected It4rAbstract(@NonNull DarumaMobile dmfObject) {
        this.dmf = dmfObject;
    }

    public abstract void venderItem(String descricao, String valor, String Codigo);

    public abstract void encerrarVenda(String valorTotal, String formaPagamento);

    public abstract void configurarXmlNfce() throws InterruptedException;
}
